### PR TITLE
Fix app interaction while Dialog is open

### PR DESCRIPTION
**Pull request by danji90**
# How to

The new MUI Dialog (LayerInfo, TopicInfo) has some conflicts with certain DOM Elements while the Dialog is visible.
- Select Dropdowns (Language, Projection) can be clicked to open and hover styles apply, but the menu can't be toggled anymore, doesn't close and click on a menu option has no effect.

Since we are switching many components to MUI components and we already have an SBBSelect, the dropdowns can be solved by using the SBBSelect for all DD-Menus

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [x] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] The title means something for a human being.
- [x] The title contains [WIP] if it's necessary.
- [x] Labels applied. if it's a release? a hotfix?
- [ ] Tests added.

### CODE REVIEW

**Comment by danji90** [comment]
@oterral _disableEnforceFocus: true_ seems to have done the trick

**Review by oterral — CHANGES_REQUESTED** [review]
It seems there is duplicated styles in langSelect and projectionSelect, can we put them in the generic Select component ?

**Review by oterral — CHANGES_REQUESTED** [review]
- margin between icon and text are not the same for login text and language box
- language box is not aligned with login text

**Comment by danji90** [comment]
@oterral LGTM?

File: src/components/Select/Select.js [hunk 1]
```diff
@@ -1,67 +1,83 @@
...
+  return {
+    outlineHidden: {
+      '&:hover': {
+        color: theme.palette.secondary.dark,
+      },
+      '&:hover .MuiOutlinedInput-notchedOutline,& .MuiOutlinedInput-notchedOutline':
+        {
+          borderWidth: 0,
+        },
+      '& Mui-focused .MuiOutlinedInput-notchedOutline': {
```
Comment (oterral) [inline src/components/Select/Select.js:16]: & .Mui-focused

Comment (oterral) [inline src/components/Select/Select.js:16]:
```suggestion
      '& .Mui-focused .MuiOutlinedInput-notchedOutline': {
```

**Comment by oterral** [comment]
@danji90  there is BAFU layers change in this PR is it normal ?

**Review by oterral — APPROVED** [review]
Very nice thanks, Ihave fixed the border of the select box on mobile

**Pull request merged**